docs/Toast/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Installation
 
-Backpack Android is available through [Jitpack](https://jitpack.io/#Skyscanner/backpack-android). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+Backpack Android is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-android). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
```

Comment:
Nice catch! We should probably remove this section eventually or use some sort of snipper for that.

Reply:
yeah would be nice if we could essentially merge md files here, rather than having to specify this for each page